framework/axSGreaseSkeleton.js

```diff
@@ -53,9 +106,9 @@ function setAriaIdIfNecessary(elem) {
 	return elem.id;
 }
 
-function makeElementOwn(parentElement, listOfNodes){
+function makeElementOwn(parentElement, listOfNodes) {
```

Comment:
For future reference, I'd prefer to keep completely unrelated white space fixes in separate patches, just to make "real" patches easier to follow. Just leave these ones though; it's all good.

Reply:
I use vscode, which fixes indent as part of linting (alt+shift+f or format-on-save if configured). What do you use to avoid these kind of changes from happening at work? My code, all being under format-on-save, never accumulates random errors with indent like this.

Reply:
I'm very old fashioned and still use Notepad++. The Mozilla build system (and the NVDA build system) has commands to run various linters, which I always execute before submitting things. I've tried on multiple occasions to adopt VS Code, but I just can't get to a point where it helps my efficiency/productivity overall, rather than hindering it.